lib/puppet/provider/package/homebrew.rb

```diff
@@ -52,11 +54,16 @@ def self.package_list(options={})
         unless result.include? name
           result += execute([command(:brew), :cask, :list, '--versions', name])
         end
+        if result.empty?
```

Comment:
this is just to show debug if the package was found or not. Should we remove this part?